pkg/sfu/rtpmunger.go

```diff
@@ -197,6 +197,15 @@ func (r *RTPMunger) UpdateAndGetSnTs(extPkt *buffer.ExtPacket) (*TranslationPara
 	if diff < 0 {
 		// out-of-order, look up sequence number offset cache
 		snOffset, err := r.snRangeMap.GetValue(extPkt.ExtSequenceNumber)
+		r.logger.Debugw(
+			"out-of-order packet",
+			"extHighestIncomingSN", r.extHighestIncomingSN,
+			"extLastSN", r.extLastSN,
+			"extSequenceNumber", extPkt.ExtSequenceNumber,
+			"snOffset", snOffset,
+			"error", err,
+			"outgoingSN", extPkt.ExtSequenceNumber-snOffset,
+		)
```

Comment:
other debug logs added in this PR can remain, but this could be noisy. Will remove after debug.